### PR TITLE
Initial dockerfile

### LINKED ISSUE #17: Provide Docker image

### PR DESCRIPTION
**Pull request by rootwarp**
close #17

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -13,3 +12,10 @@ Copy **session ID** and try below.
 ```
 ~$ ./index.js ceremony <session id>
 ```
+
+You can also use docker.
+
+```
+~$ docker run rootwarp/czg-keramony:latest index.js auth
+~$ docker run rootwarp/czg-keramony:latest index.js <session id>
+```
```
Comment (kim201212) [inline README.md]: Sorry, There is a typo on the repo name.
Please fix the name of the docker image.
`czg-keramony` -> `czg-keremony`

File: README.md [hunk 2]
```diff
@@ -1,6 +1,5 @@
 # Czg-Keramony
```
Comment (kim201212) [inline README.md]: Please fix the typo 🙏 
`Czg-keramony` -> `CZG-Keremony`

Comment (rootwarp) [reply README.md]: Fixed 1286aea83986fa66360b20fee7a44472af5dc0b1

**Comment by kim201212** [comment]
When I test to docker, The below error appears.

```
    data: {
      code: 'ContributeError::NotUsersTurn',
      error: 'not your turn to participate'
    }
```

It's slower than when I tested it at Local.
It takes a long time.

**Comment by rootwarp** [comment]
@kim201212 
Running platform changed. arm64 -> amd64

**Review by kim201212 — APPROVED** [review]
It is working well! 
LGTM!

**Pull request merged**